### ISSUE TITLE
[YOUTUBE MUSIC WEBSITE APP]

### ISSUE THREAD

**Issue by saffff1** [web-scrobbler/web-scrobbler#4598]
I use youtube music and when i use the app it doesn't fill in the song information. 

![song info] (https://github.com/web-scrobbler/web-scrobbler/assets/137027606/50abb2c8-974e-49c5-8a90-0215f2324584)

![yt music app](https://github.com/web-scrobbler/web-scrobbler/assets/137027606/9ce54f74-8fc9-446d-a1e2-1762e168f88f)

{update} 3/16/2024
it starts workin after i play a few songs.

**Comment by yayuyokitano** [comment]
Probable duplicate of #4571 
See if this issue still occurs in 3.7.0 once it is approved by extension stores.

**Comment by rob9315** [comment]
works nowadays
<details><summary>screenshot</summary>
<img width="1265" height="805" alt="Image" src="https://github.com/user-attachments/assets/5ae72b9d-2e19-422d-bc4f-2641f4ab2133" />
</details>
